server/__tests__/api.test.js

```diff
@@ -217,6 +217,15 @@ describe('tls', () => {
     ]);
   });
 
+  it('Check /tls/tls-version with date filter', async () => {
+    const res = await request(app).get(
+      '/api/tls/tls-version?year=2018&month=04',
```

Comment:
I think the approach of this test should be the following:

- Try with a different year
- Try with a different month

You must adapt the server mock to read those parameters and respond accordingly.

I know this might be difficult to implement, so it's just a recommendation.

Reply:
Thanks! I think I'll try this in a different PR

Reply:
Great! No problem